src/buildtool.ts

```diff
@@ -6,7 +6,7 @@ export interface RunBuildOptions {
 }
 
 export function generateCommand(options: RunBuildOptions): string {
-    let command: string = "buildtool";
+    let command: string = "addpath('"+ process.cwd() +"/plugins');buildtool";
```

Comment:
Single quotes need to be escaped here. I'm also not sure if `addpath` will treat '/' as a file separator on Windows. Probably best to use `fullfile` instead.

Reply:
My get removed as David's comments

Reply:
Done changes with `__dir`

Reply:
Single quotes still need to be escaped.